server/mux.go

```diff
@@ -367,18 +369,21 @@ func NewMux(opts MuxOpts, service Service) http.Handler {
 	var out http.Handler
 
 	/* Authentication */
-	if opts.UseAuth {
+	if opts.BasicAuth != nil {
```

Comment:
If `BasicAuth` and `UseAuth` are both specified, does `BasicAuth` take precedence? Feels like it should be reversed.

Reply:
Yes, it does now. It is not expected that both OATH and BasicAuth are used at the same time.

Reply:
but if I do specify both, which one wins?

Reply:
basicAuth now wins, np to make OAuth win if you think that it would be better

Reply:
Lets go with the oauth winning :)